backend/util/posts.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function () {
+  // All tags and their count, where keys are the tag values and the values are the number of posts using that tags.
+  let allTags = {};
+
+  // The process for plotting all posts via a Python script.
+  // TODO
+  const spawn = require("child_process").spawn;
+  const createPlotPosts = spawn("python3", []);
+
+  /***
+   * Plots the point values of all posts.
+   * Runs once--upon starting the server.
+   */
+  async function plotPosts() {
+    let allPosts = await prisma.post.findMany();
```

Comment:
It's generally recommended to not iterate through a list that you're mutating. You can create a new array and push the new object in each iteration of the loop

Reply:
Got it; thanks!